scalapb-runtime/src/test/scala/scalapb/FieldMaskTreeSpec.scala

```diff
@@ -0,0 +1,34 @@
+package scalapb
+
+import com.google.protobuf.field_mask.FieldMask
+import munit.FunSuite
+
+class FieldMaskTreeSpec extends FunSuite {
+  test("testAddFieldPath") {
+    // https://github.com/protocolbuffers/protobuf/blob/v3.6.0/java/util/src/test/java/com/google/protobuf/util/FieldMaskTreeTest.java#L39-L62
```

Comment:
The version used for testing is fairly outdated, can you catch up the tests with the 3.14.x branch?

Reply:
Yes, I noticed it, but didn't want to touch other tests in `FieldMaskUtilsSpec`. I will update all them to `3.14` then